vitrina/datasets/templates/vitrina/datasets/list.html

```diff
@@ -54,7 +54,13 @@
     {% if request.resolver_match.url_name == 'organization-datasets' %}
         {% include 'vitrina/orgs/tabs.html' %}
     {% endif %}
-
+    {% if view.kwargs.kind and view.kwargs.slug %}
+        {% if user.is_authenticated and user.organization.slug == view.kwargs.slug %}
```

Comment:
```suggestion
    {% if can_create_dataset %}
```

Move business logic out of templates and use same `can_create_dataset` service function as in views, to ensure consistency. Also make sure, that `user.is_staff` can create new datasets.